.github/workflows/lint_test_build.yml

```diff
@@ -46,6 +46,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: ./.github/workflows/rafiki/env-setup
       - run: pnpm --filter openapi build
+      - run: pnpm --filter auth fetch-schemas
```

Comment:
Actually, I wonder if `fetch-schemas` also needs to be run in the Dockerfile
https://github.com/interledger/rafiki/blob/main/packages/auth/Dockerfile#L18

Reply:
Yeah, local env fails as is with:
```
local-peer-auth-1     | {"level":50,"time":1667568836819,"pid":1,"hostname":"f5cbb380f8b7","msg":"ResolverError: Error opening file \"/workspace/openapi/auth-server.yaml\" \nENOENT: no such file or directory, open '/workspace/openapi/auth-server.yaml'\n    at ReadFileContext.callback (/workspace/node_modules/.pnpm/@apidevtools+json-schema-ref-parser@9.0.9/node_modules/@apidevtools/json-schema-ref-parser/lib/resolvers/file.js:52:20)\n    at FSReqCallback.readFileAfterOpen [as oncomplete] (node:fs:323:13)"}
```
I'm thinking instead of adding `fetch-schemas` to the Dockerfile, we just add it to https://github.com/interledger/rafiki#environment-setup
That way localenv can run with modified specs.